addons/survey/populate/survey_survey.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools import populate
+
+class Survey(models.Model):
+    _inherit = 'survey.survey'
+    _populate_sizes = {'small': 10, 'medium': 50, 'large': 100}
+
+    def _populate_factories(self):
+        random = populate.Random('survey_survey')
+        highest_code = int(self._read_group([], [], ['session_code:max'])[0][0])
+        user_ids = self.env['res.users'].search([]).ids
+        user_range = [_ for _ in range(0, 11)]
```

Comment:
`n_survey_users_options` ?
Also `0`, is default start.
```suggestion
        n_survey_users_options = [_ for _ in range(11)]
```